src/database/seeders/20190911110843-destination.js

```diff
@@ -0,0 +1,45 @@
+export default {
+  up: async (queryInterface) => queryInterface.bulkInsert('Destinations', [
+    {
+      destination: 'Lagos',
+    },
+    {
+      destination: 'Kampala',
+    },
+    {
+      destination: 'Cairo',
+    },
+    {
+      destination: 'Texas',
+    },
+    {
+      destination: 'Paris',
+    },
+    {
+      destination: 'Saudi Arabia',
+    },
+    {
+      destination: 'United Arab Emirates',
+    },
+    {
+      destination: 'Singapore',
+    },
+    {
+      destination: 'Singapore',
+    },
+    {
+      destination: 'Singapore',
+    },
+    {
+      destination: 'Russia',
```

Comment:
I noticed repetition of Russia in the seeder is this on purpose

Reply:
Yes it is...